dammit/tasks.py

```diff
@@ -168,7 +168,7 @@ def get_lastdb_task(db_fn, db_out_prefix, lastdb_cfg, prot=True):
     exc = which('lastdb')
     params = lastdb_cfg['params']
     if prot:
-        params += ' -p'
+        params += ' -p -w3'
```

Comment:
I've added this to `dammit/.config.json` already, so this should be removed (also, the placement here only applies it for protein databases :))